code/implementation/docker/machine/src/main/resources/schema/base/amazonec2Config.json.d/amazonec2.json

```diff
@@ -12,8 +12,7 @@
         },
         "vpcId": {
             "type": "string",
```

Comment:
LGTM if we decide to move the validation part to the UI, or decide that we don't need it at all. @will-chan ?